modules/era/lua_dynamis/mobs/era_xarcabard_mobs.lua

```diff
@@ -249,7 +255,7 @@ xi.dynamis.onFightDynaLord = function(mob, target)
         for key, var in pairs(dwagonVars) do -- Update Ying and Yang to Attack Current Target
             if mob:getZone():getLocalVar(key) == 0 then
                 local dwagon = GetMobByID(var[1])
-                if not dwagon:isEngaged() then
+                if dwagon and not dwagon:isEngaged() then
                     dwagon:updateEnmity(target)
```

Comment:
Do you mind getting rid of dwagon and making it an actual name while you are here?

Reply:
> Do you mind getting rid of dwagon and making it an actual name while you are here?

Yep, I changed to just dragon to avoid confusion when readying code.